src/main/webapp/app/components/geneHistoryTooltip/gene-history-tooltip-utils.tsx

```diff
@@ -108,9 +108,31 @@ export function formatLocation(location: string, drugList: IDrug[], objectField:
       return location + ', Description of Evidence';
     } else if (objectField === 'oncogenic') {
       return location + ', Oncogenic';
+    } else if (objectField === 'pathogenic') {
+      return location + ', Pathogenicity';
     }
   } else if (lastSubstring.endsWith('Tumor Type Summary')) {
     return locationSubstrings.slice(0, -1).join(',') + ', Therapeutic Summary';
+  } else if (lastSubstring.endsWith('Mutation Specific Cancer Risk')) {
```

Comment:
It would depend. I would assume the title would still include the words `Cancer Risk`. In that case, the check could be updated just to look for those words. If the title did not include those words, we would be forced to use the same location going forward (or at least the words `Cancer Risk`).